README.md

```diff
@@ -16,6 +16,42 @@ Program defines what arguments are expected.
 `kotlinx.cli` will figure out how to parse those, reporting errors if the program arguments are invalid,
 and also generate help and usage messages as well.
 
+## Using in your projects
+
+> Note that the library is experimental and the API is subject to change.
+
+The library is published to [kotlinx](https://bintray.com/kotlin/kotlinx/kotlinx.cli) bintray repository.
+
+### Gradle
+
+Add the bintray repository:
+
+```groovy
+repositories {
+    maven { url "https://kotlin.bintray.com/kotlinx" }
+}
+```
+
+In multiplatform projects add the following dependency to the common source set:
+
+```groovy
+kotlin {
+    sourceSets {
+        commonMain {
+             dependencies {
+                 implementation("org.jetbrains.kotlinx:kotlinx-cli:0.2.1")
+             }
+        }
+    }
+}
+```
+
+To use the library in a JVM-only project add the platform to the artifact name, e.g.:
+
+```groovy
+implementation("org.jetbrains.kotlinx:kotlinx-cli-jvm:0.2.1")
```

Comment:
AFAIK, it is possible to specify the umbrella MPP artifact name in JVM-only projects too, Gradle then selects jvm variant automatically.

Reply:
Perhaps should be in `dependencies` block.

Reply:
So without `jvm`? Then this part can be just removed and it would be enough to write in needed source set.

Reply:
I don't think there's `kotlin { sourceSets { } }` block and `commonMain` source set in JVM . But maybe it would be enough to have just an mpp example.